src/test/fuzz/coins_view.cpp

```diff
@@ -150,8 +150,7 @@ void TestCoinsView(FuzzedDataProvider& fuzzed_data_provider, CCoinsView& backend
                         coins_cache_entry.coin = *opt_coin;
                     }
                     auto it{coins_map.emplace(random_out_point, std::move(coins_cache_entry)).first};
-                    if (dirty) CCoinsCacheEntry::SetDirty(*it, sentinel);
-                    if (fresh) CCoinsCacheEntry::SetFresh(*it, sentinel);
+                    if (dirty) CCoinsCacheEntry::SetDirty(*it, sentinel, fresh);
```

Comment:
definition of `fresh` should likely include `dirty` now:
```C++
const auto fresh{dirty && fuzzed_data_provider.ConsumeBool()};
```

Reply:
Done.